collect_app/src/main/java/org/odk/collect/android/instancemanagement/send/InstanceUploaderListActivity.java

```diff
@@ -381,6 +381,7 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long row
             long instanceId = c.getLong(c.getColumnIndex(DatabaseInstanceColumns._ID));
             Intent intent = FormFillingIntentFactory.editDraftFormIntent(this, projectsDataService.requireCurrentProject().getUuid(), instanceId);
             startActivity(intent);
+            finish();
```

Comment:
This will also return the user to the main menu after viewing a form which is a change in behaviour. I can imagine this being annoying in a flow where you want quickly check forms before you send them, and I don't think we should mess with it.

Reply:
As mentioned in the description, when you open a form from the list of sent forms and then close it, you're taken back to the main menu. So why not apply the same behavior to the list of forms to send? It makes the experience more consistent.